Source/ACE.Server/Managers/PlayerManager.cs

```diff
@@ -122,6 +131,16 @@ public static OfflinePlayer GetOfflinePlayer(uint guid)
             return null;
         }
 
+        public static int GetOfflineCount()
+        {
+            return offlinePlayers.Count;
+        }
+
+        public static int GetOnlineCount()
```

Comment:
This is not thread safe. The proper way to get all online players is: PlayerManager.GetAllOnline().Count

This is how it's done in "serverstatus"

Reply:
modified this function to use the correct method